cfgov/v1/util/filterable_list.py

```diff
@@ -116,3 +115,9 @@ def form_id(self):
             return form_ids[0]
         else:
             return 0
+
+    def serve(self, request):
+        """ Modify response header to set a shorter TTL in Akamai """
+        response = super(FilterableListMixin, self).serve(request)
+        response['edge-control'] = 'cache-maxage=10m'
```

Comment:
We've enabled the use of etags as well as Last-modified. It's less than ideal, because we still need to render the document before doing the etag comparison. 

Doing that better, was part of what I was getting at in my last paragraph on GHE/CFGOV/platform/issues/550#issuecomment-106643

We should do revalidation better, but that's a seperate concern.